pkg/security/probe/activity_dump.go

```diff
@@ -911,6 +941,15 @@ func (pan *ProcessActivityNode) snapshotFiles(p *process.Process, ad *ActivityDu
 	return nil
 }
 
+// InsertDNSEvent inserts
+func (pan *ProcessActivityNode) InsertDNSEvent(evt *model.DNSEvent) bool {
+	if _, ok := pan.DNSNames[evt.Name]; ok {
+		return false
+	}
+	pan.DNSNames[evt.Name] = NewDNSNode(evt)
```

Comment:
Aggregate by event type (A/AAA) and record type

Reply:
👍🏻